oak_crypto/src/encryptor.rs

```diff
@@ -47,6 +50,12 @@ impl EncryptionKeyProvider {
         }
     }
 
+    pub fn new_from_keys(private_key: PrivateKey, public_key: PublicKey) -> Self {
```

Comment:
Preexisting: I think the `new` function could be renamed to `random` (or `generate`) and `new_from_keys` to just `new`